KubeArmor/core/k8sHandler.go

```diff
@@ -287,15 +287,31 @@ func (kh *K8sHandler) PatchResourceWithAppArmorAnnotations(namespaceName, deploy
 		return nil
 
 	} else if kind == "Pod" {
-		_, err := kh.K8sClient.CoreV1().Pods(namespaceName).Patch(context.Background(), deploymentName, types.MergePatchType, []byte(spec), metav1.PatchOptions{})
+		// this condition wont be triggered but keeping here for reference
+
+		pod, err := kh.K8sClient.CoreV1().Pods(namespaceName).Get(context.TODO(), deploymentName, metav1.GetOptions{})
```

Comment:
Why are we doing a Get here?

Reply:
cannot patch pods 

so here we are getting the pod 
adding annotation and then updating the pods

Reply:
this is not triggered 
should i just remove this ?